lib/workers/repository/update/pr/automerge.ts

```diff
@@ -121,6 +127,12 @@ export async function checkAutoMerge(
       prAutomergeBlockReason: PrAutomergeBlockReason.DryRun,
     };
   }
+  if (GlobalConfig.get('dryRun') === 'lookup') {
+    return {
+      automerged: false,
+      prAutomergeBlockReason: PrAutomergeBlockReason.DryRun,
+    };
+  }
```

Comment:
```suggestion
```

If dry-run is lookup, we should never be in the "update" logic here in the first place